implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -162,7 +162,7 @@ pub struct GlobalArgs {
     #[arg(global = true, long, short, default_value_t = quiet_default_value())]
     quiet: bool,
 
-    /// Increase verbosity of trace messages
+    /// Increase verbosity of trace messages by repeating the flag. Learn more at https://crates.io/crates/clap-verbosity-flag”
```

Comment:
@rjtch this is super helpful improvement since people might not know how to increase the verbosity. But I think instead of linking out to the crate, it would be better to instead describe the repetition and its effect right here in the help.

Reply:
Hey @mrinalwadhwa I have just changed the docs according to your comment!!